tests/_pyMylib/__init__.py

```diff
@@ -0,0 +1,31 @@
+import ctypes
+from pathlib import Path
+
+from threadpoolctl import LibController, register
+
+
+path = Path(__file__).parent / "my_threaded_lib.so"
+ctypes.CDLL(path)
+
+
+class MyThreadedLibController(LibController):
+    user_api = "my_threaded_lib"
+    internal_api = "my_threaded_lib"
+    filename_prefixes = ("my_threaded_lib",)
```

Comment:
```suggestion
    # names for threadpoolctl's context filtering
    user_api = "my_threaded_lib"
    internal_api = "my_threaded_lib"

    # patterns to identify the name of the linked library to load
    # If a library with a matching filename is linked to the python
    # process, it will be available as a dynlib object in the LibController.
    filename_prefixes = ("my_threaded_lib",)
```

Reply:
@tomMoral I edited your suggestion directly :)